app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala

```diff
@@ -26,12 +26,13 @@ import uk.gov.hmrc.tariffclassificationfrontend.models.Case
 import uk.gov.hmrc.tariffclassificationfrontend.utils.JsonFormatters.caseFormat
 
 import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
 
 @Singleton
 class AuditService @Inject()(auditConnector: DefaultAuditConnector) {
 
   def auditCaseReleased(c: Case)
-                       (implicit hc: HeaderCarrier): Unit = {
+                       (implicit hc: HeaderCarrier): Future[Unit] = {
```

Comment:
Why `Future[Unit]`
The type is `Unit`
See here: https://github.com/hmrc/play-auditing/blob/master/src/main/scala/uk/gov/hmrc/play/audit/http/connector/AuditConnector.scala

Reply:
For consistency. All the connectors we use have a `Future` return type, this should be no different. Under the hood it is a thread blocking HTTP request so sticking it in a future means we can parallelise it.
Also improves the Service layer code having it as a future

Reply:
reverted